packages/zoe/test/unitTests/zcf/test-offerHandlerStorage.js

```diff
@@ -2,18 +2,19 @@
 
 // eslint-disable-next-line import/no-extraneous-dependencies
 import { test } from '@agoric/zoe/tools/prepare-test-env-ava';
+import { Far } from '@agoric/marshal';
 
 import { makeOfferHandlerStorage } from '../../../src/contractFacet/offerHandlerStorage';
 
 test('offerHandlerStorage', async t => {
   const { storeOfferHandler, takeOfferHandler } = makeOfferHandlerStorage();
 
-  const offerHandler = () => {};
+  const offerHandler = Far('offerHandler', () => {});
```

Comment:
Why `Far` here and not the new `ToFarFunction`?

Reply:
Added to `ToFarFunction` a clarification that it is for use with functions you get from others. For functions written right here, use `Far` directly on the function literal. Resolving.